build.gradle

```diff
@@ -80,14 +80,15 @@ dependencies {
     compile "org.hibernate:hibernate-core:5.1.2.Final"
     compile "org.hibernate:hibernate-ehcache:5.1.2.Final"
     compile 'org.grails.plugins:spring-security-core:3.0.4'
-    compile "org.grails.plugins:mail:2.0.0"
+    compile "org.grails.plugins:mail:3.0.0"
     compile "org.grails.plugins:grails-spring-websocket:2.4.1"
     compile "eu.bitwalker:UserAgentUtils:1.21"
 
     // runtime
     runtime "com.h2database:h2"
     runtime 'mysql:mysql-connector-java:5.1.36'
     runtime "com.bertramlabs.plugins:asset-pipeline-grails:2.11.6"
+    runtime 'javax.xml.bind:jaxb-api:2.3.0'
```

Comment:
what do you need this library for?

Reply:
run streama with java 11 will failed for `javax.xml.bind` not found .

for details:
java 9 remove javax.xml.bind from jre, still can be added as module;
java 11 totally remove javax.xml.bind from jre.